app/Repositories/DefaultAnimeRepository.php

```diff
@@ -124,7 +126,16 @@ public function getAiredBetween(
 //        ]);
 
         /** @noinspection PhpParamsInspection */
-        $queryable = $this->queryable(true)->whereRaw([
+        $queryable = $this->queryable(true);
+
+        if ($premiered !== null) {
+            $queryable = $this->queryable()
+                ->where("premiered", null)
+                ->orWhere("premiered", $premiered);
+        }
+
+        $queryable = $this->queryable()
+            ->whereRaw([
```

Comment:
`$queryable` is all over the place here.

```php
$queryable = $this->queryable(true);
```
This is a completely new query builder.

Then if `$premiered` is null, we overwrite it's value with the instance level query builder. We should consolidate this.

The difference between
```php
$queryable = $this->queryable(true);
```
and
```php
$queryable = $this->queryable();
```
is that whether we want to chain the previous query builder or not. E.g you use `$queryable = $this->queryable()` because you just want to append `where` clauses to the existing query builder.

Reply:
@pushrbx Good catch. It should be re-using `$queryable` - was doing this in a hurry 😅

Reply:
@pushrbx should be good now